js/test/Ice/info/Client.ts

```diff
@@ -47,11 +47,6 @@ export class Client extends TestHelper {
         test(ipEndpoint.sourceAddress == "10.10.10.10");
         test(ipEndpoint.compress);
         test(!ipEndpoint.datagram());
-        test(
```

Comment:
As far as I can tell this section of test still needs to be removed from C#, Java, and Python. Do we really want to remove this tests though? I don't think we removed the API.